libraries/abstractions/common_io/include/iot_i2c.h

```diff
@@ -247,8 +320,9 @@ int32_t iot_i2c_write_sync( IotI2CHandle_t const pxI2CPeripheral,
  * Partial read might happen, e.g. slave device unable to receive more data.
  * And the number of bytes that have been actually read can be obtained by calling iot_i2c_ioctl.
  *
- * @Note Usually, the address of register needs to be written before calling this function.
- * @Note If eI2CSendNoStopFlag is set and this function returns, whether the actual transaction has been started is undefined. This is board-specific behavior.
+ * @note Usually, the address of register needs to be written before calling this function.
+ * @note If eI2CSendNoStopFlag is set and this function returns, whether the actual transaction has been started is undefined. This is board-specific behavior.
+ * @note Usually, iot_i2c_set_callback is called prior to this in order to get notification when the asynchronous call completes.
```

Comment:
Should it say "iot_i2c_set_callback() must be called prior to calling this API in order to get notification when the asynchronous operation is completed"

Reply:
You want set_callback mandatory? I was thinking this is optional, based on what user needs.

Reply:
No. Looks like even my comments are not clear..:-)
I meant - callback must be set for notifications to happen

Reply:
Ah, my bad. OK, I will rephrase it.